src/frontend/hooks/useHentBarnetrygdbehandlinger.ts

```diff
@@ -4,12 +4,14 @@ import { useHttp } from '@navikt/familie-http';
 
 import { hentBarnetrygdbehandlinger } from '../api/hentBarnetrygdbehandlinger';
 
-export const BARNETRYGDBEHANDLINGER_QUERY_KEY_PREFIX = 'barnetrygdbehandlinger';
+export const HentBarnetrygdbehandlingerQueryKeyFactory = {
+    fagsak: (fagsakId: number | undefined) => ['klagebehandlinger', fagsakId],
```

Comment:
Trenger egentlig denne å ta i mot `undefined`?

```suggestion
    fagsak: (fagsakId: number) => ['klagebehandlinger', fagsakId],
```